website/database.py

```diff
@@ -585,6 +585,9 @@ def batch_get_adventures(self, adventure_ids):
         keys = {id: {"id": id} for id in adventure_ids}
         return ADVENTURES.batch_get(keys) if keys else {}
 
+    def get_public_adventures(self):
```

Comment:
Let's not forget to add this table to prod and alpha when this gets deployed

Reply:
This is the adventures table which exists on prod, right? Or does that have to be changed when we change its schema? Because I did include a new index to the table!